.github/workflows/publish.yml

```diff
@@ -43,8 +43,18 @@ jobs:
       run: |
         m="${{github.event.inputs.site_mode}}"
         if [ "$m" = "production" ]; then
+          echo "BUCKET_URL=s3://api2static.onlyoffice.com" >> "$GITHUB_ENV"
+          echo "AWS_ACCESS_KEY_ID=${{ secrets.ACCESS_KEY_PROD }}" >> "$GITHUB_ENV"
+          echo "AWS_SECRET_ACCESS_KEY=${{ secrets.SECRET_KEY_PROD }}" >> "$GITHUB_ENV"
+          echo "AWS_REGION=us-west-2" >> "$GITHUB_ENV"
+          echo "AWS_DISTRIBUTION_ID=${{ secrets.AWS_DISTRIBUTION_ID_PROD }}" >> "$GITHUB_ENV"
           /usr/bin/time --verbose pnpm build:prod
         elif [ "$m" = "testing" ]; then
+          echo "BUCKET_URL=s3://api2static.teamlab.info" >> "$GITHUB_ENV"
+          echo "AWS_ACCESS_KEY_ID=${{ secrets.ACCESS_KEY }}" >> "$GITHUB_ENV"
+          echo "AWS_SECRET_ACCESS_KEY=${{ secrets.SECRET_KEY }}" >> "$GITHUB_ENV"
+          echo "AWS_DISTRIBUTION_ID=${{ secrets.AWS_DISTRIBUTION_ID }}" >> "$GITHUB_ENV"
+          echo "AWS_REGION=us-east-1" >> "$GITHUB_ENV"
```

Comment:
Please, move the environment setup in a separate step.

```yaml
- name: Setup AWS Environment
  run: |
    if
```

Reply:
Was make in 798b289